packages/vite-plugin-checker/package.json

```diff
@@ -46,8 +46,7 @@
     "commander": "^8.0.0",
     "fast-glob": "^3.2.7",
     "fs-extra": "^11.1.0",
-    "lodash.debounce": "^4.0.8",
-    "lodash.pick": "^4.4.0",
+    "lodash": "^4.17.21",
```

Comment:
Could we use `lodash-es` instead as it works better with ESM.

Reply:
@nclavaud but still leave import from js I guess. I've tested it with ESM when testing ".js" extension and it still was bad in tests

Reply:
I think it's way better to import 

```js
import { pick } from 'lodash-es'
```

instead of

```js
import pick from 'lodash-es/pick.js'
```

In ESM, extension is required when importing from a file.

Reply:
@fi3ework sure, I just pushed another commit